pkg/credentialprovider/plugin/examples/acr/login.go

```diff
@@ -0,0 +1,285 @@
+// +build !providerless
```

Comment:
This file was copied from https://github.com/kubernetes/kubernetes/blob/master/pkg/credentialprovider/azure/azure_acr_helper.go